test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py

```diff
@@ -33,20 +37,55 @@ def sanitizeFolderPath(self, folderPath):
         return folderPath.rstrip("/")
 
     def addAccount(self, context):
-        server, user, password, localfolder = getClientDetails(context)
+        self.addServer(context)
+        self.addUserCredentails(context)
+        try:
+            squish.clickButton(squish.waitForObject(self.ERROR_OK_BUTTON))
+        except LookupError:
+            pass
+        self.changeSyncDirectory(context)
+        self.connectAccount()
+
+    def addServer(self, context):
+        server = getClientDetails(context)[0]
 
         squish.mouseClick(squish.waitForObject(self.SERVER_ADDRESS_BOX))
         squish.type(squish.waitForObject(self.SERVER_ADDRESS_BOX), server)
         squish.clickButton(squish.waitForObject(self.NEXT_BUTTON))
-        squish.mouseClick(squish.waitForObject(self.SERVER_ADDRESS_BOX))
+
+    def addUserCredentails(self, context):
+        _, user, password, _ = getClientDetails(context)
+
         squish.type(squish.waitForObject(self.USERNAME_BOX), user)
         squish.type(squish.waitForObject(self.USERNAME_BOX), "<Tab>")
         squish.type(squish.waitForObject(self.PASSWORD_BOX), password)
         squish.clickButton(squish.waitForObject(self.NEXT_BUTTON))
-        try:
-            squish.clickButton(squish.waitForObject(self.ERROR_OK_BUTTON))
-        except LookupError:
-            pass
+
+    def selectFoldersForSync(self, context):
+        self.openSyncDialog()
+
+        squish.mouseClick(
+            squish.waitForObjectItem(self.CHOOSE_WHAT_TO_SYNC_CHECKBOX, "/"),
+            11,
+            11,
```

Comment:
Do you have to click on that co-ordinate specifically?

Reply:
Yeah, That's the co-ordinate for the checkbox.
I could not get the element for the checkbox only.